recipes/pauda-mergeblast/merge.blast.pl

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env perl
+# xiechaos@gmail.com
```

Comment:
What I mean is you are downloading this script from http://ab.inf.uni-tuebingen.de/data/software/pauda/download/merge.blast.pl ... why is this included in the PR as single file and you copy it not to PREFIX/bin, isn't it?